refbox/src/refBox.js

```diff
@@ -67,7 +67,18 @@ const AVAILABLE_TEAMS = [
   'Tech United Eindhoven',
   'Hibikino Musashi',
   'er@sers',
-  'Tiny Boy',
+]
+
+
+const AVAILABLE_CHALLENGES = [
+  'Cocktail party',
+  'Restaurant',
+]
+
+
+const AVAILABLE_ATTEMPTS = [
+  '1',
+  '2',
```

Comment:
I guess this will be a config file (yaml or json) for the server later on right? Because this shouldn't be hardcoded

Reply:
Yes, I planned on doing that in a different feature but forgot to add the issue. See #17